src/details/ArborX_DetailsMinimumSpanningTree.hpp

```diff
@@ -357,10 +357,9 @@ void updateLowerBounds(ExecutionSpace const &space, Labels const &labels,
   Kokkos::parallel_for(
       "ArborX::MST::update_lower_bounds",
       Kokkos::RangePolicy<ExecutionSpace>(space, 0, n), KOKKOS_LAMBDA(int i) {
-        using KokkosExt::max;
```

Comment:
FWIW I liked slightly better the previous version with an using-declaration.

Reply:
We are in control of the data it is operating on here, so I don't think it matters.